internal/distroregistry/distroregistry_test.go

```diff
@@ -65,11 +65,6 @@ func TestRegistry_mangleHostDistroName(t *testing.T) {
 		{"rhel-84 stream", args{"rhel-84", false, true}, "rhel-8"},
 		{"rhel-84 beta stream", args{"rhel-84", true, true}, "rhel-8-beta"},
 
-		{"centos-8", args{"centos-8", false, false}, "centos-8"},
-		{"centos-8 beta", args{"centos-8", true, false}, "centos-8-beta"},
-		{"centos-8 stream", args{"centos-8", false, true}, "centos-stream-8"},
```

Comment:
CentOS Stream 8 is not EOL. Only the CentOS 8, which was based on rebuilds of released RHEL-8 minor versions.